base_report_to_label_printer/models/res_users.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2022 Raumschmiede GmbH - Christopher Hansen (<https://www.raumschmiede.de>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    default_label_printer_id = fields.Many2one(
+        comodel_name="printing.printer", string="Default Label Printer"
+    )
+
+    @api.model
+    def _register_hook(self):
+        super()._register_hook()
+        self.SELF_WRITEABLE_FIELDS.extend(["default_label_printer_id"])
```

Comment:
should override `SELF_READABLE_FIELDS` and  `SELF_WRITEABLE_FIELDS` property https://github.com/odoo/odoo/blob/09fa1db600c31d2d04f0504f0412cbbc85ff7e3d/odoo/addons/base/models/res_users.py#L351